src/createApp.ts

```diff
@@ -22,16 +23,21 @@ export default async (): Promise<Koa> => {
   app.proxy = true
 
   app
+    .use(morgan('common'))
+    .use(helmet())
+    .use(errorHandler(error))
+    .use(
+      serve(path.resolve(__dirname, '..', 'apidoc'), {
```

Comment:
How about mounting the doc serving to `/apidoc` or `/doc` routes rather serving it to root url.

Reply:
Done